### PR TITLE
bpo-32731: Specify Exception type of getpass.getuser

### PR DESCRIPTION
**Pull request by fangyi-zhou**
https://bugs.python.org/issue32731

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept your contribution by verifying you have signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

Unfortunately our records indicate you have not signed the CLA. For legal reasons we need you to sign this before we can look at your contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.


We also demand... [A SHRUBBERY!](https://www.youtube.com/watch?v=zIV4poUZAQo)


Thanks again to your contribution and we look forward to looking at it!

**Comment by fangyi-zhou** [comment]
I've signed CLA, how do I ask the bot to check again?

**Review by matrixise — APPROVED** [review]
Thank you for your PR, the wording seems fine.

File: Misc/NEWS.d/next/Documentation/2018-02-11-03-23-36.bpo-32731.eHIUEe.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+Improve documentation of exception raised in :func:`getpass.getuser()`
```
Comment (brettcannon) [inline Misc/NEWS.d/next/Documentation/2018-02-11-03-23-36.bpo-32731.eHIUEe.rst:1]:
```suggestion
Improve documentation of exceptions raised in :func:`getpass.getuser()`.
```

File: Doc/library/getpass.rst [hunk 2]
```diff
@@ -43,9 +43,10 @@ The :mod:`getpass` module provides two functions:
...
    :envvar:`USER`, :envvar:`LNAME` and :envvar:`USERNAME`, in order, and
-   returns the value of the first one which is set to a non-empty string.  If
-   none are set, the login name from the password database is returned on
-   systems which support the :mod:`pwd` module, otherwise, an exception is
-   raised.
+   returns the value of the first one which is set to a non-empty string.
+   If none are set, the function attempts to retrieve the login name from the
+   password database. On systems which the :mod:`pwd` module is not supported,
+   an :exc:`ImportError` is raised. If no login name is found, an
+   :exc:`KeyError` is raised :func:`pwd.getpwuid`.
```
Comment (brettcannon) [inline Doc/library/getpass.rst:50]: What is the mention for `pwd.getwuid()` for? Is it to state that it's the function that causes `KeyError` to be raised?

And are there tests that these exception are raised in these scenarios?

**Review by brettcannon — CHANGES_REQUESTED** [review]

**Pull request closed without merging**